constants/src/lib.rs

```diff
@@ -2,6 +2,7 @@ pub const VENDOR_IDENTIFIER: &str = "MongoDB";
 pub const DRIVER_NAME: &str = "MongoDB Atlas SQL interface ODBC Driver";
 pub const DBMS_NAME: &str = "MongoDB Atlas";
 pub const ODBC_VERSION: &str = "03.80";
+pub const DEFAULT_APP_NAME: &str = "odbc-driver";
```

Comment:
The default app name I see for the JDBC driver is `mongodb-jdbc+{version}`.
What do you think about keeping things similar and use`mongodb-odbc+{version}`?
For example `mongodb-jdbc+2.0.2`.
I think adding the version is worth it for future data collection. It can be useful sometime to know if users are upgrading their driver or stay with older versions and/or see if there are users using older version when a new version with breaking changes is released.

Reply:
Good point!

Reply:
I've added version information for appName, as long as the user hasn't specified their own appName @nbagnard . Thanks for the suggestion!